ouroboros-consensus-shelley/test/Test/Consensus/Shelley/Ledger/Golden.hs

```diff
@@ -54,9 +60,63 @@ tests = testGroup "Golden tests"
     , testCase "ApplyTxErr"     test_golden_ApplyTxErr
     , testCase "ConsensusState" test_golden_ConsensusState
     , testCase "LedgerState"    test_golden_LedgerState
+    , testCase "HeaderState"    test_golden_HeaderState
+    , testCase "ExtLedgerState" test_golden_ExtLedgerState
+    , testQueries
       -- TODO Query and result
     ]
 
+testQueries :: TestTree
+testQueries = testGroup "Queries"
+    [ testCase "LedgerTip"
+        $ test_golden_Query GetLedgerTip tipTerm
+    , testCase "EpochNo"
+        $ test_golden_Query GetEpochNo echoNoTerm
+    , testCase "NonMyopicMemberRewards"
+        $ test_golden_Query (GetNonMyopicMemberRewards Set.empty) myopicMemberRewardsTerm
```

Comment:
We should include at least one element in the set, so that if that type changes, we'd notice it.